doc/custom-metrics-adapter.md

```diff
@@ -51,12 +51,14 @@ External metrics allow you to autoscale your cluster based on any metric availab
 ```
 
  * metric_name: The name of metric generated by OAL or other subsystem.
- * label: available `label_key`s are `service` and `label`. 
+ * label: `label_key` is from the arguments of swctl . 
   
- The options of label keys are:
- * `service` varies in different scopes. If the scope is `Service`, `ServiceInstance` and `Endpoint`, the `service` represents
-    `serviceName`, `serviceInstanceName` and `endpointName` respectively.
- * `label` is optional, used for querying multi-labels metrics.
+The options of label keys are:
+ * `service` The name of the service.
+ * `instance` The name of the service instance.
+ * `endpoint` The name of the endpoint.
+ * `label` is optional, The labels you need to query, used for querying multi-labels metrics. Unlike [swctl](https://github.com/apache/skywalking-cli#metrics-multiple-linear), 
```

Comment:
We could use only one label here, right?

Reply:
exactly. k8s api doesn't support querying more than one metric, so I only introduce a single label.